### ISSUE TITLE
Update runtimes

### ISSUE THREAD

**Issue by Anchor5178** [flathub/io.github.Figma_Linux.figma_linux#27]
```
Info: org.freedesktop.Sdk//20.08 is end-of-life, with reason:
   org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: org.freedesktop.Platform//20.08 is end-of-life, with reason:
   org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Applications using this runtime:
   io.github.Figma_Linux.figma_linux
Info: org.freedesktop.Platform.GL.default//20.08 is end-of-life, with reason:
   org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
```

**Comment by MTRNord** [comment]
Any update on this? :)

**Comment by hfiguiere** [comment]
this was closed with #29
